tiledb/cloud/dag/dag.py

```diff
@@ -895,6 +899,33 @@ def submit_udf(self, func, *args, **kwargs):
 
     submit = submit_udf
 
+    def submit_udf_stage(
+        self, func, *args, expand_node_output: Optional[Node] = None, **kwargs
+    ):
+        """
+        Submit a function that will be executed in the cloud serverlessly
+        :param func: function to execute
+        :param args: arguments for function execution
+        :param expand_node_output: the Node that we want to expand the output of. The output of the node should be a
```

Comment:
For `expand_node_output`, can we support multiple parameters being passed or complex structures? Does it have to be JSON encoded? Can we use arrow or native serialization?

Reply:
I think that Argo can only expand one node output. In this version, we don't have support for passing more complex structures (i.e. pickled values). However, I think that this is achievable by adding a custom serialization to the UDF images in order to have an output of a JSON list of TGArgs